playground/src/experiments/experiments.js

```diff
@@ -0,0 +1,147 @@
+// Copyright 2020 The AMPHTML Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import './experiments.scss';
+import template from './experiments.hbs';
+import experimentListItem from './experiment-list-item.hbs';
```

Comment:
If you decouple the template it would be nice to also decouple the logic, similar to error-list-item.js